pkg/datapath/loader/base.go

```diff
@@ -161,6 +167,24 @@ func cleanIngressQdisc(devices []string) error {
 	return nil
 }
 
+func linkList() ([]netlink.Link, error) {
```

Comment:
Please add a comment here and in the commit message explaining the reason for the retries.

Reply:
Done